community-expectations.md

```diff
@@ -0,0 +1,83 @@
+## Community Expectations
+
+Kubernetes is a community project. Consequently, it is wholly dependent on
+its community to provide a productive, friendly and collaborative environment.
+
+The first and foremost goal of the Kubernetes community to develop orchestration
+technology that radically simplifies the process of creating reliable
+distributed systems. However a second, equally important goal is the creation
+of a community that fosters easy, agile development of such an orchestration
+systems.
+
+We the community therefore describe the expectations for
+members of the Kubernetes community.  This document is intended to be a living one
+that evolves as the community evolves via the same PR and code review process
+that shapes the rest of the project.  It currently covers the expectations
+of conduct that govern all members of the community as well as the expectations
+around code review that govern all active contributors to Kubernetes.
+
+### Code of Conduct
+
+The most important expectation of the Kubernetes community is that all members
+abide by the Kubernetes [community code of conduct](code-of-conduct.md).
+Only by respecting each other can we develop a productive, collaborative
+community.
+
+### Code review
+
+As a community we believe in the value of code review for all contributions.
+Code review increases both the quality and readability of our codebase, which
+in turn produces high quality software.
+
+However, the code review process can also introduce latency for contributors
+and additional work for reviewers that can frustrate both parties.
+
+Consequently, as a community we expect that all active participants in the
+community will also be active reviewers.
+
+Any *active contributor* to the Kubernetes project is expected to be available
+to review code in areas where that contributor has expertise.  Active
+contributors are considered to be anyone who meets any of the following criteria:
+   * Sent more than two pull requests (PRs) in the previous one month, or more
+   than 20 PRs in the previous year.
+   * Filed more than 3 issues in the previous month, or more than 30 issues in
+   the previous year.
+   * Commented on more than 5 issues or pull requests in the previous month, or
```

Comment:
Maybe I'll make this "Commented on more than 5 PRs?"  I think if you are actively commenting on PRs then you are by definition an active reviewer and should be considered for the pool.

Also there are several people in the project who do a lot of reviewing and commenting, but often don't actually write any PRs.